surfer/viz.py

```diff
@@ -1292,6 +1296,12 @@ def add_annotation(self, annot, borders=True, alpha=1, hemi=None,
             alpha_vec = cmap[:, 3]
             alpha_vec[alpha_vec > 0] = alpha * 255
 
+            # Override the cmap when a single color is used
+            if color is not None:
+                from matplotlib.colors import colorConverter
+                rgb = np.round(np.multiply(colorConverter.to_rgb(color), 255))
+                cmap[:, :3] = rgb.astype(cmap.dtype)
```

Comment:
Should it be `:4` and `to_rgba`?

Reply:
There's already a parameter in `add_annotation` that controls the opacity of the overlay. I thought it made the most sense to continue honoring that argument and to ignore any alpha channel in the color spec itself.

In other words, like this

```
b.add_annotation("aparc", color="forestgreen", alpha=.5)
```
![annot_edges](https://user-images.githubusercontent.com/315810/62884073-a3131880-bd03-11e9-892f-45b0cff0aabb.png)

I guess one could check to see if the color has an alpha channel and error or warn if it does not match the alpha specified in the function call. But seems like overkill?